modules/ROOT/pages/create-instance-task.adoc

```diff
@@ -9,20 +9,29 @@ You can create one instance of an API that serves as a proxy. You can create ano
 
 After setting up an API version for management, you repeat the process to create another instance. Attempting to set up the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.
 
-To create a API instance:
+To create an API instance:
 
-. Select *API Administration > Manage API* to create an API instance from Exchange.
+. Select *API Administration* > *Manage API* > *Create a new API*.
 +
-For example, to set up version 1.0 of an API named `jsonplaceholder`:
+For example, set up version 1.0 of an API named `jsonplaceholder`:
 +
 image::manage-exchange-end-task.png[]
 +
 . From *API Name*, select the API (`jsonplaceholder`).
-. From *API Version*, select version (`1.0`).
-. From *Asset Version*, select the Exchange asset version.
+. From *Asset types*, select the type of API specification that you want to create.
++
+[Note] OAS 3.0 specifications are currently not supported in API Manager. You must convert the API to RAML or OAS 2.0 to manage using API Manager.
++
+. If you are creating an API type other than HTTP, upload the specification file in the *File upload* field.
```

Comment:
Upload of Asset is only required if they are using the "Create New API" flow. If they are managing from Exchange, user is searching on their Exchange Assets.